### ISSUE TITLE
Communicate with WCAG 2.2 community participants

### ISSUE THREAD

**Issue by calvin-lau-sig7** [alphagov/govuk-design-system#3430]
## What
Following publish of WCAG 2.2 guidance #3405, we want to tell our decision making rationale to design sprint research participants and workshop attendees.

Currently drafting in [comms plan Google doc](https://docs.google.com/document/d/1idlkUtopSuqGMXOwSsHVH3_oAvCK7lxX1mewZopDySo/edit?usp=sharing)

## Why
To build on the relationship with people that have helped guide us in WCAG 2.2 work.

## Who needs to work on this
Calvin, Charlotte

## Who needs to review this
Delivery manager, Product manager, Communiter manager

## Done when
- [ ] draft our decisions, rationale, next steps and open questions
- [ ] send via email
- [ ] link to message on Slack channels (optional)
- [ ] create discussions on GitHub, if we want to use that to open any questions
